src/components/course/data/hooks.jsx

```diff
@@ -187,13 +187,13 @@ export function useCourseRunWeeksToComplete(courseRun) {
 
   useEffect(() => {
     // this conditional will always fail at courseRun.weeksToComplete === 0, because it always equals false
-    if (courseRun && courseRun.weeksToComplete) {
+    // adding additional logical operator to allow a zero value for weeks to complete to pass the conditional
+    if (courseRun && (courseRun.weeksToComplete || courseRun.weeksToComplete >= 0)) {
```

Comment:
nit: 

Is:

```
if (courseRun && (courseRun.weeksToComplete || courseRun.weeksToComplete >= 0)) {
```

essentially equivalent to:

```
if (courseRun?.weeksToComplete >= 0) {
```

Reply:
Yep. Finally a perfect use case for it.